core/src/test/scalajs/sttp/client3/testing/AbstractFetchHttpTest.scala

```diff
@@ -27,6 +27,8 @@ abstract class AbstractFetchHttpTest[F[_], +P] extends HttpTest[F] {
 
   override protected def supportsHostHeaderOverride = false
 
+  override protected def supportsCharsetControl = false
```

Comment:
This should probably say sth about multiparts, e.g. `supportsCharsetsInMultipartParts`